### PR TITLE
Fix the description of index_url in /python/package-index/state

### PR DESCRIPTION
**Pull request by VannTen**
## Related Issues and Dependencies

…

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Fix the description of index_url in /python/package-index/state

I'm pretty sure that index_url is not a secret, given the rest of the endpoint
and the register one.

### CODE REVIEW

File: openapi/openapi.yaml [hunk 1]
```diff
@@ -1506,7 +1506,7 @@ paths:
         - name: index_url
           in: query
           required: true
-          description: A secret to register the given Python package index.
+          description: URL to the Python simple repository as described in PEP 503.
```
Comment (mayaCostantini) [inline openapi/openapi.yaml:1509]:
```suggestion
          description: URL to the Python package index to register.
```
Here `https://pypi.org/simple` is the default value

**Review by mayaCostantini — COMMENTED** [review]
Thanks for spotting this! Just one comment

File: openapi/openapi.yaml (hunk 1 above)
Comment (VannTen) [reply openapi/openapi.yaml:1509]: It's not on the register endpoint though, but in the state one (enabling/disabling the index); I reused the description from the register endpoint (in the request body, `properties.url`)

So maybe `package index to enable or disable` ?

Comment (mayaCostantini) [reply openapi/openapi.yaml:1509]: Correct :+1:

**Pull request merged**